buddyboss/buddyboss-platform
# 979 - fixed translate his/her/their determiners

### All Submissions:






### Changes proposed in this Pull Request:



Fixes https://github.com/buddyboss/buddyboss-platform/issues/979 

### How to test the changes in this Pull Request:

1. Enable the gender profile field.
2. Choose your Gender.
3. Change the Profile photo.
4. Check Activity.

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
